src/platform/mhv/api/mocks/index.js

```diff
@@ -33,7 +33,7 @@ const acceleratedCareSummariesAndNotes = require('./medical-records/care-summari
 const healthConditions = require('./medical-records/health-conditions');
 const acceleratedHealthConditions = require('./medical-records/health-conditions/accelerated');
 const allergies = require('./medical-records/allergies');
-const acceleratedAllergies = require('./medical-records/allergies/full-example');
+const acceleratedAllergies = require('./medical-records/allergies/accelerated');
```

Comment:
Suggestion: To help with local testing, let's keep the mock data for OH Patient eg.
`const allergiesOH = require('./medical-records/allergies/full-example')`